src/modules/Library/features/AppletsCatalog/AppletsCatalog.tsx

```diff
@@ -79,8 +75,8 @@ export const AppletsCatalog = () => {
             {t('appletsCatalog')}
           </StyledHeadlineLarge>
           <StyledAppletList>
-            {publishedApplets?.data?.length
-              ? publishedApplets.data
+            {publishedApplets?.result?.length
+              ? publishedApplets.result
                   // TODO: delete slice when endpoint is ready
```

Comment:
do we still need this todo?

Reply:
yes. pagination is still not ready on a backend.